cmd/bootconfig/main.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 The NATS Authors
```

Comment:
I wonder if at some point (not this PR) a special build of the NATS server into a process that includes the sidecars/helpers might be cleaner.

Reply:
I like this idea a lot...